src/vec.cpp

```diff
@@ -27,6 +27,12 @@ using namespace std;
 
 namespace meep {
 
+static bool _wholenumber(double value) {
```

Comment:
I would just call it
```cxx
static bool isinteger(double x) { return x == floor(x); }
```
(We haven't generally been using underscores for `static` functions, since those are automatically private.)